README.md

```diff
@@ -67,6 +67,23 @@ Tools may introduce their own data nodes with custom keys and values. Other tool
 
 This witness specification is a work in progress and will be subject to modifications.
 
+
+### Additional Edge Data for Concurrent Programs
+
+Validating concurrent programs is a complex task,
+because we need to know possible interleavings of thread operations and when a thread is started or exited.
+The following information should additionally be available in the witness:
+
+| key | Meaning | Allowed in Violation Witnesses | Allowed in Correctness Witnesses |
+| --- | --- | ---- | ---- |
+| threadId | Represents the currently active thread for the edge. If no ``threadId`` is given, any thread can be active. The value should be a unique identififer for a thread. | Yes | Yes |
+| createThread | The currently active thread creates a new thread. Using a ``threadId`` is only allowed after creating a matching thread. The thread's function can be entered after this edge, using the ``enterFunction`` key in combination with the ``threadId`` of the created thread. | Yes | Yes |
```

Comment:
Could you please explain more clearly that this ``createThread`` tag's value is the id of the created thread, that a ``threadId`` on a transition with this tag refers to the thread that "triggers" the creation, and that the subsequent transition you are referring to (which should, as you mention, use the corresponding ``enterFunction`` tag) then should have as a value of its ``threadId`` tag the id of the created tag?